_posts/2022-04-25-tardigrade-launch.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title:  "ETL in Trino speeds with project Tardigrade: now ready for early users"
+author: Andrii Rosa, Brian Zhan, Lukasz Osipiuk, Zebing Lin
+excerpt_separator: <!--more-->
+---
+
+After 6 months of challenging work, we’ve launched Project Tardigrade. It is designed to improve user experience of running resource intensive queries that are common in ETL space. It’s available right now! It required some significant and fascinating engineering to get us here. Read on to learn how Project Tardigrade works and how to enable it. We think you’ll enjoy hearing about it.
+
+<!--more-->
+
+## What is Project Tardigrade
+
+What we love most about doing ETL with Trino is that it’s fast. It lets you use the same dialect between interactive and ETL and it allows you iterate fast through intuitive error messages, interactive experience, and query federation.
+
+If you’ve done large scale ETL processing before, you probably had to deal with following problems:
+
+* Some queries may need to spend thousands of cpu / hours and multiple hours of runtime on pre-processing large data sets what makes them wasteful or sometimes even impossible (without breaking an SLA) to restart in an event of a failure. 
```

Comment:
suggest re-write to (for better clarity):

* Reliable landing times: Queries that run for hours can fail and restart from scratch. This wastes CPU resources and makes it hard for you to meet your data landing SLA.